tests/h/search/index_test.py

```diff
@@ -84,7 +122,7 @@ def test_it_marks_annotation_as_deleted(self, es):
             index='hypothesis',
             doc_type='annotation',
             body={'deleted': True},
-            id='test_annotation_id'
+            id='test_annotation_id',
```

Comment:
Same point as above about style-only changes mixed in with functional changes. In this case, I’d even say this change isn’t worth the extra work it would cause for someone looking through the file with `git annotate`.